docs/stdlib-times.md

```diff
@@ -72,6 +72,7 @@ times := import("times")
   2006" would be interpreted if it were the value; it serves as an example of
   the input format. The same interpretation will then be made to the input
   string.
+- `parseInLocation(format string, s string, loc string)`: parseInLocation is like parse but parse interprets a time as UTC; parseInLocation interprets the time as in the given location.
```

Comment:
Here I think we can write all arguments as `parseInLocation(format, s, loc string)`

Reply:
True, but none of the other functions in this file are written like that